src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -0,0 +1,147 @@
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
+import React, {useRef} from 'react';
+import lodashGet from 'lodash/get';
+import {compose} from 'underscore';
+import PropTypes from 'prop-types';
+import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsDefaultProps, withCurrentUserPersonalDetailsPropTypes} from '../../components/withCurrentUserPersonalDetails';
+import ONYXKEYS from '../../ONYXKEYS';
+import FullPageNotFoundView from '../../components/BlockingViews/FullPageNotFoundView';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import TabSelector from '../../components/TabSelector/TabSelector';
+import CONST from '../../CONST';
+import useLocalize from '../../hooks/useLocalize';
+import * as IOUUtils from '../../libs/IOUUtils';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import MoneyRequestAmount from './steps/MoneyRequestAmount';
+import ReceiptSelector from './ReceiptSelector';
+import * as IOU from '../../libs/actions/IOU';
+import Tab from '../../libs/actions/Tab';
+import DragAndDropProvider from '../../components/DragAndDrop/Provider';
+
+const TopTab = createMaterialTopTabNavigator();
+
+const propTypes = {
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            iouType: PropTypes.string,
+            reportID: PropTypes.string,
+        }),
+    }),
+
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        currency: PropTypes.string,
+        participants: PropTypes.arrayOf(
+            PropTypes.shape({
+                accountID: PropTypes.number,
+                login: PropTypes.string,
+                isPolicyExpenseChat: PropTypes.bool,
+                isOwnPolicyExpenseChat: PropTypes.bool,
+                selected: PropTypes.bool,
+            }),
+        ),
+    }),
+
+    /** Which tab has been selected */
+    selectedTab: PropTypes.string,
+
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+const defaultProps = {
+    route: {
+        params: {
+            iouType: '',
+            reportID: '',
+        },
+    },
+    iou: {
+        id: '',
+        amount: 0,
+        currency: CONST.CURRENCY.USD,
+        participants: [],
+    },
+    selectedTab: CONST.TAB.TAB_MANUAL,
+    ...withCurrentUserPersonalDetailsDefaultProps,
+};
+
+function MoneyRequestSelectorPage(props) {
+    const iouType = useRef(lodashGet(props.route, 'params.iouType', ''));
+    const {translate} = useLocalize();
+
+    const isEditing = useRef(lodashGet(props.route, 'path', '').includes('amount'));
```

Comment:
I don't think we need `isEditing` here. This is only needed for the amount page.